### ISSUE TITLE
Gulp

### ISSUE THREAD

**Issue by sayus2884** [sayus2884/cam#1]
Convert to Gulp

**Comment by iRaySpace** [comment]
Are you aiming for automated task? I think you can do it with Grunt by using Grunt Watch.

**Comment by sayus2884** [comment]
True. You can do that in grunt but if possible, I want it to be more readable to others for them to understand the process more. Also, more plugins will be added in the future which means that maintaining the configs will become a hassle. Though, right now, grunt will have to suffice since there are only a few things in the file.
